apps/ledger-live-desktop/src/newArch/features/Collectibles/hooks/useCollectiblesAnalytics.ts

```diff
@@ -0,0 +1,41 @@
+import { track } from "~/renderer/analytics/segment";
+
+export enum AnalyticsPage {
```

Comment:
it's not the responsibility of a hook to export enums, i recommend to extract this to a local `enums.ts` file